### PR TITLE
Fix CVE-2022-21698, CVE-2022-27664, Upgrade to Go1.18

### PR DESCRIPTION
**Pull request by mrproliu**

### CODE REVIEW

**Comment by wu-sheng** [comment]
Welcome back.

**Review by wu-sheng — APPROVED** [review]

File: docs/en/guides/compile/How-to-compile.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Go version
 
-Go version `1.17` or higher is supported for compilation.
+Go version `1.18` or higher is supported for compilation.
```
Comment (kezhenxu94) [inline docs/en/guides/compile/How-to-compile.md:5]: Can you confirm this is correct? Usually I found higher Go version cannot compile lower Go version project seamlessly, usually some modifications are needed

Comment (mrproliu) [reply docs/en/guides/compile/How-to-compile.md:5]: I have checked the GO 1.19.2, It works well.
image

Comment (wu-sheng) [reply docs/en/guides/compile/How-to-compile.md:5]: Let's add 1.19 in the build matrix, and only declare we support these two major versions.

**Review by wu-sheng — DISMISSED** [review]

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**